File: MyProject/ForTest/app/src/main/java/com/kelfan/fortest/Util/FileHandler.java
package com.kelfan.fortest.Util;


import com.kelfan.fortest.Constant;

/**
 * Created by Administrator on 15/01/2018.
 */

public class FileHandler {

    public static String read_app_file(String filename) {
        return FileWorker.readStringFromSecondSD(Constant.DEFAULT_FILE_PATH, filename);
    }

    public static int write_app_file(String filename, String txt) {
        return FileWorker.writeToSecondSD(Constant.DEFAULT_FILE_PATH, filename, txt);
    }

    public static int append_app_file(String filename, String txt) {
        return FileWorker.appendToSecondSD(Constant.DEFAULT_FILE_PATH, filename, txt);
    }

}

File: MyProject/07KotlinEditor/app/src/main/java/com/kelfan/kotlineditor/util/FileHandler.kt
package com.kelfan.kotlineditor.util

import android.content.Context

/**
 * Created by Administrator on 21/01/2018.
 */

class FileHandler {
    companion object{
        fun read_data_file(filename: String): String {
            return FileWorker.readStringFromSecondSD(Constant.DEFAULT_FILE_PATH, filename)
        }

        fun write_data_file(filename: String, txt: String): Int {
            return FileWorker.writeToSecondSD(Constant.DEFAULT_FILE_PATH, filename, txt)
        }

        fun read_app_file(context: Context, filename: String): String {
            val file:String = FileWorker.getAppPath(context, Constant.DIRECTORY_DATA)!!
            return FileWorker.readAllText(file)
        }
    }
}